packages/reporter/src/test/test.tsx

```diff
@@ -192,15 +192,13 @@ class Test extends Component<TestProps> {
       )
     }
 
-    if (appState.studioActive) {
-      controls.push(
-        <Tooltip key={`studio-command-${this.props.model}`} placement='right' title='Add Commands to Test' className='cy-tooltip'>
-          <a onClick={this._launchStudio} className='runnable-controls-studio'>
-            <WandIcon />
-          </a>
-        </Tooltip>,
-      )
-    }
+    controls.push(
+      <Tooltip key={`studio-command-${this.props.model}`} placement='right' title='Add Commands to Test' className='cy-tooltip'>
+        <a onClick={this._launchStudio} className='runnable-controls-studio'>
+          <WandIcon />
+        </a>
+      </Tooltip>,
+    )
```

Comment:
Note for `_launchStudio` below:

I wasn't able to get studio to launch unless I clicked the icon, navigated away from the runner and then back in. I refactored the function to set the `studioActive` property and it started working

```ts
_launchStudio = (e: MouseEvent) => {
    e.preventDefault()
    e.stopPropagation()

    const { model, events } = this.props

    action('studio:init:test', () => {
      appState.setStudioActive(true)
      events.emit('studio:init:test', model.id)
    })()
}
```

Reply:
Good catch - I'd been testing with a spec containing `it('is empty', () => {})`. I'd missed this case. I'll fix it.